meta/documents/changelog_de.md

```diff
@@ -7,6 +7,8 @@
 - Es ist nicht mehr möglich, ein Kontaktformular für plentyShop zu versenden, ohne die Datenschutzrichtlinien zu akzeptieren.
 - In der mobilen Ansicht ist die permanent eingeblendete Suche nicht mehr durch Darstellungsfehler beeinträchtigt.
 - In der mobilen Ansicht kann die Spracherkennung wieder geschlossen werden, die durch das Widget **Spracherkennung** eingebunden wird.
+- In der mobilen Ansicht werden die Bilder im **Artikelraster** Widget mit der Einstellung "Artikelanzahl pro Zeile (Mobil): 2"  wieder korrekt dargestellt.
```

Comment:
```suggestion
- In der mobilen Ansicht werden die Bilder im **Artikelraster** Widget mit der Einstellung **Artikelanzahl pro Zeile (Mobil): 2**  wieder korrekt dargestellt.
```
**suggestion (non-blocking): Formatting**

All UI elements should be in bold. Doesn't matter if it's the name of a widget, a label or a setting.